client/blocks/app-banner/utils.js

```diff
@@ -13,36 +13,49 @@ export const ONE_MONTH_IN_MILLISECONDS = 2419200000; // 28 days
 export const TWO_WEEKS_IN_MILLISECONDS = 1209600000;
 export const ONE_DAY_IN_MILLISECONDS = 86400000;
 
-export function getAppBannerData( translate, sectionName ) {
+export function getAppBannerData( translate, sectionName, isRTL ) {
 	switch ( sectionName ) {
 		case GUTENBERG:
 			return {
 				title: translate( 'Rich mobile publishing.' ),
 				copy: translate(
 					'A streamlined editor with faster, simpler image uploading? Check and mate.'
 				),
+				icon: isRTL
+					? '/calypso/animations/app-promo/wp-to-jp-rtl.json'
+					: '/calypso/animations/app-promo/wp-to-jp.json',
```

Comment:
```suggestion
				icon: `/calypso/animations/app-promo/wp-to-jp${ isRTL ? '-rtl' : '' }.json`
```

Just an idea to make this slightly less verbose :) (would also work below)

Reply:
Ooh, nice! Thank you! I've gone ahead to implement your suggestion for all the lines in https://github.com/Automattic/wp-calypso/pull/67013/commits/0f6ba29bad9f7149d4476946f2557d584eb1aca7.